aspnet-core/xml/ns-Microsoft.AspNetCore.Hosting.Server.Abstractions.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Hosting.Server.Abstractions">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides the <see cref="T:Microsoft.AspNetCore.Hosting.Server.Abstractions.IHostContextContainer[T]" /> type that can be implemented by a server to pool <see cref="T:Microsoft.AspNetCore.Http.HttpContext" /> instances.</summary>
```

Comment:
What's the proper type syntax for generics?

Reply:
I believe it's 
`<xref:Microsoft.AspNetCore.Hosting.Server.Abstractions.IHostContextContainer%601>` or
` <xref:Microsoft.AspNetCore.Hosting.Server.Abstractions.IHostContextContainer%601?displayProperty=nameWithType>`

to display name with type.

Reply:
```suggestion
    <summary>Provides the <see cref="T:Microsoft.AspNetCore.Hosting.Server.Abstractions.IHostContextContainer%601" /> type that can be implemented by a server to pool <see cref="T:Microsoft.AspNetCore.Http.HttpContext" /> instances.</summary>
```